distribution/docs/src/main/resources/content/_managing/_running/other-ingest.adoc

```diff
@@ -3,7 +3,7 @@
 :status: published
 :summary: Other methods of ingesting data.
 :parent: Ingesting Data
-:order: 04
+:order: 05
 
```

Comment:
Is the order for `Ingesting Data` still changed?

Reply:
yes, there were two that were `03` previously